pipelines/py/bundle.yaml

```diff
@@ -0,0 +1,36 @@
+name: Install Python packages with UV
+
+needs:
+  packages:
+    - python-${{inputs.python-version}}-base
+    - uv
+
+inputs:
+  dependencies:
+    description: "Indirect dependencies to install."
+    default: ""
+  package:
+    description: "Name of packages to install."
+    required: true
+  python-version:
+    description: "Python version to use."
```

Comment:
How about a python path instead? Looks like `UV_PYTHON` supports that. It would be more consistent with our other python pipelines, and would lend itself to venvs in cases where that's necessary.